### PR TITLE
tweaks to functions for readability / performance

### PR DESCRIPTION
**Pull request by michaelrosesandow**

### CODE REVIEW

File: stimulus/erlang.py [hunk 1]
```diff
@@ -8,12 +8,13 @@ def intensity(aht, interval, rate):
 def erlang_b(server_count, intensity):
     ib = 1.0
     server_count = int(server_count) # need explicit cast here to prevent range TypeError
-    for i in range(0, server_count):
+    for i in xrange(0, server_count):
```
Comment (m-deck) [inline stimulus/erlang.py:12]: I'm wondering the best way to do this since I'd like Stimulus to be PY3 compatible...

File: stimulus/erlang.py [hunk 2]
```diff
@@ -26,24 +27,39 @@ def service_level(server_count, rate, interval, aht, wait_time):
...
 
-def required_server_count(target, rate, interval, aht):
+def validate_sl_target(t):
+    if 0.0 < t <= 1.0:
+        raise ValueError('SL must be between 0 and 1')
+
+    return True
+
+def validate_asa_target(t):
+    if t < 0:
```
Comment (m-deck) [inline stimulus/erlang.py]: <=

File: stimulus/erlang.py [hunk 3]
```diff
@@ -8,12 +8,13 @@ def intensity(aht, interval, rate):
...
     server_count = int(server_count) # need explicit cast here to prevent range TypeError
-    for i in range(0, server_count):
+    for i in xrange(0, server_count):
         ib = 1.0 + ib * (i / intensity)
     return 1.0 / ib
 
 def erlang_c(server_count, intensity):
-    return server_count * erlang_b(server_count, intensity) / (server_count - intensity * (1 - erlang_b(server_count, intensity)))
+    erl_b = erlang_b(server_count, intensity)
+    return server_count * erl_b / (server_count - intensity * (1 - erl_b))
```
Comment (m-deck) [inline stimulus/erlang.py:18]: 👍 nice!

**Review by m-deck — CHANGES_REQUESTED** [review]
Thanks for your contribs! Left a few minor comments.

File: stimulus/erlang.py (hunk 1 above)
Comment (michaelrosesandow) [reply stimulus/erlang.py:12]: `from past.builtins import xrange`

**Review by m-deck — APPROVED** [review]
thankya

**Pull request merged**